lib/remoteproc/remoteproc.c

```diff
@@ -521,7 +521,7 @@ int remoteproc_load(struct remoteproc *rproc, const char *path,
 			img_data = NULL;
 			/* get the I/O region from remoteproc */
 			pa = METAL_BAD_PHYS;
-			(void)remoteproc_mmap(rproc, &pa, &da, nmemsize, 0, &io);
+			remoteproc_mmap(rproc, &pa, &da, nmemsize, 0, &io);
```

Comment:
this is to avoid compilation warning in some system.

Reply:
Fix